### PR TITLE
LocationHierarchy Inventory Filter

### LINKED ISSUE #81: Only download service points with inventory
Currently, on the app, all the service points are downloaded and the user can view all. This number is quite large. To reduce the number of service points users can download and view, improve performance and meet the user requirement as a monitoring tool for supplies, there is a need to filter out all the service points and leave only those with inventory within the accountability period. 

**Required**
- Implement filtering of service points by inventory. 

**Acceptance criteria**
- [ ] Only service points with valid inventory are downloaded and seen on the app register and map views when a user selects a region and district.

### PR DESCRIPTION
**Pull request by lincmba**
fixes #81

**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Resolves [link to issue]

**Engineer Checklist**

- [ ] I have written **Unit tests** for any new feature(s) and edge cases for
      bug fixes
- [ ] I have added documentation for any new feature(s) and configuration
      option(s) on the `README.md`
- [ ] I have run `mvn spotless:check` to check my code follows the project's
      style guide
- [ ] I have run `mvn clean test jacoco:report` to confirm the coverage report
      was generated at `plugins/target/site/jacoco/index.html`
- [ ] I ran `mvn clean package` right before creating this pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -396,6 +396,31 @@ Example:
 [GET] /LocationHierarchy?_id=<some-location-id>&administrativeLevelMin=2&administrativeLevelMax=4&_count=<page-size>&_page=<page-number>&_sort=<some-sort>
 ```
 
+##### Inventory Filters
+
+The `LocationHierarchy` endpoint supports filtering by inventory status,
```
Comment (dubdabasoduba) [inline README.md:401]:
```suggestion
The `LocationHierarchy` endpoint supports filtering by inventory availability,
```

Comment (dubdabasoduba) [inline README.md:401]:
```suggestion
The `LocationHierarchy` endpoint supports filtering by inventory availability,
```

**Review by dubdabasoduba — APPROVED** [review]

**Review by dubdabasoduba — APPROVED** [review]
